setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(name='postnl_api',
-      version='1.0.2',
+      version='2.0.0',
```

Comment:
Could you bump to 1.1.0? It doesn't look like there are any breaking changes.
Please also update the CHANGELOG.md.